plugins/woocommerce/src/Blocks/BlockTypes/AddToCartForm.php

```diff
@@ -246,10 +284,60 @@ protected function render( $attributes, $content, $block ) {
 			)
 		);
 
+		if ( $is_stepper_style ) {
+			$context = array();
+			if ( $product->is_type( 'grouped' ) ) {
+				$context['groupedProductIds']   = array();
+				$context['quantityConstraints'] = array();
+				foreach ( $product->get_children() as $child_product_id ) {
+					$child_product = wc_get_product( $child_product_id );
+					if ( $child_product && $child_product->is_purchasable() && $child_product->is_in_stock() && ! $child_product->has_options() ) {
+						$context['groupedProductIds'][] = $child_product_id;
+						$min                            = apply_filters( 'woocommerce_quantity_input_min', $child_product->get_min_purchase_quantity(), $child_product );
```

Comment:
Heads-up that every time we call `apply_filters()`, it needs to be accompanied by a doc block, ideally containing `@since 10.1.0`, which is the version where this hook will be included here.